src/index.html

```diff
@@ -4,9 +4,333 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#" class="header__logo">
+        <img src="images/logo.png" alt="На главную">
+      </a>
+      <nav class="header__nav-bar">
+        <ul class="header__nav-list">
+          <li class="header__nav-item">
+            <a href="#" class="header__nav-link is-active">
+            Apple
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a href="#" class="header__nav-link">
+            Samsung
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a href="#" class="header__nav-link">
+              Смартфоны и телефоны
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a
+            href="#"
+            class="header__nav-link"
+            data-qa="nav-hover">
+              Ноутбуки и компьютеры
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a href="#" class="header__nav-link">
+            Гаджеты
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a href="#"  class="header__nav-link">
+            Планшеты
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a href="#" class="header__nav-link">
+            Фото
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a href="#" class="header__nav-link">
+            Видео
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main-section">
+      <div class="product-card" data-qa="card">
+        <div class="product-card__image-placeholder">
+          <img src="images/imac.jpeg"
+          alt="iMac 27"
+          class="product-card__image">
+        </div>
+        <h1 class="product-card__title">
+          Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+        </h1>
+        <p class="product-card__product-code">
+          Код товара: 195434
+        </p>
+        <div class="product-card__reviews">
+          <div class="product-card__stars">
+            <div class="product-card__star_active"></div>
+            <div class="product-card__star_active"></div>
+            <div class="product-card__star_active"></div>
+            <div class="product-card__star_active"></div>
+            <div class="product-card__star"></div>
+          </div>
+          <div class="product-card__reviews_text">
+            Отзывов: 5
+          </div>
+        </div>
+        <div class="product-card__price-tag">
+          <div class="product-card__price-text">Цена:</div>
+          <div class="product-card__price-amount">69 999 грн
+          </div>
+        </div>
+        <a
+        href="#"
+        class="product-card__link"
+        data-qa="hover">
+        КУПИТЬ
+        </a>
```

Comment:
```suggestion
        <a
          href="#"
          class="product-card__link"
          data-qa="hover"
        >
          Купить
        </a>
```
Тут текст пиши обычным кеглем, а через css делай uppercase